front/components/data_source/DocumentOrTableUploadOrEditModal.tsx

```diff
@@ -485,6 +488,29 @@ export function DocumentOrTableUploadOrEditModal({
                 )}
               </div>
 
+              {isTable &&
```

Comment:
This is OK for us testing but won't be really useful for large-scale testing?

Reply:
Yes, it was only for testing , with simple design - if we want to go on we'll have to decide if we want to enable it by default